### PR TITLE
cisco_1749892: add Cisco ACI to SDN plug-ins

### PR DESCRIPTION
**Pull request by bobfuru**
Added CISCO ACI integration to SDN plug-ins doc.
Related BZ [1749892](https://bugzilla.redhat.com/show_bug.cgi?id=1749892)

@sferich888 Can I please get an ack that this is OK using the external Cisco link at line 18? Request came from @mcurry-rh

### CODE REVIEW

**Comment by sferich888** [comment]
Ack - the context for this link and it source are ok for docs inclusion.

**Comment by bobfuru** [comment]
SME-approved. No QE required. 
Ready for peer review and merge. @openshift/team-documentation

File: architecture/topics/cisco.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+////
+Module included in the following assembly:
+
+* architecture/networking/network_plugins.adoc
+////
+
+The Cisco ACI CNI plug-in for {product-title} provides integration between the APIC controller and one or more {product-title} clusters connected to a Cisco ACI fabric.
```
Comment (ahardin-rh) [inline architecture/topics/cisco.adoc]: s/APIC/Cisco Application Policy Infrastructure Controller (Cisco APIC)

File: architecture/topics/cisco.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+. The Cisco ACI CNI plug-in models the entire {product-title} cluster as a VMM domain on the Cisco APIC. This provides APIC with access to the inventory of resources of the {product-title} cluster, including the number of {product-title} nodes, {product-title} namespaces, services, deployments, Pods, their IP and MAC addresses, interfaces they are using, and so on. APIC uses this information to automatically correlate physical and virtual resources in order to simplify operations.
+
+The Cisco ACI CNI plug-in is designed to be transparent for {product-title} developers and administrators and to integrate seamlessly from an operational standpoint.
+
+For more information, see 
+link:https://www.cisco.com/c/en/us/td/docs/switches/datacenter/aci/apic/white_papers/Cisco-ACI-CNI-Plugin-for-OpenShift-Architecture-and-Design-Guide.html[Cisco ACI CNI Plugin for Red Hat OpenShift Container Platform Architecture and Design Guide].
```
Comment (ahardin-rh) [inline architecture/topics/cisco.adoc]: s/Plugin/Plug-in

**Comment by ahardin-rh** [comment]
@bobfuru Just a few minor things. Otherwise, looks great!

File: architecture/topics/cisco.adoc (hunk 2 above)
Comment (bobfuru) [reply architecture/topics/cisco.adoc]: "Plugin" here is in the title and link of the third-party doc, so I'm inclined to leave their spelling in place here.

**Pull request merged**